examples/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/pulumi/pulumi-aws-native/examples
 
-go 1.20
+go 1.19
```

Comment:
Why are we changing the example's go versions? This seems like an accident.

Reply:
I just kicked off the ci-mgmt action for single native providers. I'm not sure if providers was previously using it/ keeping it up to date in the same manner as ecosystem was for bridged providers, so was hoping @lblackstone could chime in

Reply:
My understanding was that they were not, but I could be wrong. CC @danielrbradley

Reply:
It honestly doesn't look like we're that far off though, at least for this repo. Maybe worth investing a small bit of effort to get it there. It looks like we could break out a lot of the whitespace changes into a separate PR to clear some of the noise. Then we'd need to update the [sourcemigrator](https://github.com/pulumi/ci-mgmt/blob/2b5f1bcdd912fb770246d266e4717bb6fbd935cf/tools/sourcemigrator/index.ts#L63) so it doesn't downgrade the go version and then I think we could move to managing this repo's CI with this tool.